docs/tables/azuread_directory_setting.md

```diff
@@ -0,0 +1,47 @@
+# Table: azuread_directory_setting
+
+Directory settings define the configurations that can be used to customize the tenant-wide and object-specific restrictions and allowed behavior.
+
+By default, all entities inherit the preset defaults.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  display_name,
+  id,
+  values
+from
+  azuread_directory_setting;
+```
+
+### Check user admin consent workflow is enabled
+
+```sql
+select
+  display_name,
+  id,
+  values
+from
+  azuread_directory_setting
+where
+  display_name = 'Consent Policy Settings'
```

Comment:
How often are these values nested? Would it be better to pull out these values into individual rows, with the `display_name` being the same, but having `name` and `value` be individual columns?